ReactWeek1Assignment.js

```diff
@@ -0,0 +1,30 @@
+class Student {
+    constructor(name, email, community) {
+        this.name = name;
+        this.email = email;
+        this.community = community;
+    }
+}
+
+class Bootcamp {
+    constructor(name, level, students = []) {
+        this.name = name;
+        this.level = level;
+        this.students = students;
+    }
+
+    registerStudent(student) {
+        const studentAlreadyRegistered = this.students.filter(
+            (s) => s.email === student.email
+            ).length;
+
+        if (studentAlreadyRegistered) {
+            console.log(`The student with ${student.email} is already registered`);
+    } else {
```

Comment:
Everything looks good! Maybe install the prettier extension to make sure this code formatting is proper, otherwise well done 👍 

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode